test/namenode/NameNodeTest.cc

```diff
@@ -0,0 +1,110 @@
+#include "zkwrapper.h"
+#include "zk_nn_client.h"
+
+#include <gtest/gtest.h>
+
+#include <cstring>
+#include <easylogging++.h>
+
+INITIALIZE_EASYLOGGINGPP
+
+namespace {
+
+	class NamenodeTest : public ::testing::Test {
+
+	protected:
+		virtual void SetUp() {
+			system("sudo ~/zookeeper/bin/zkServer.sh stop");
```

Comment:
@nichhk When I try to move these into the main, the zookeeper calls fail with a "ZCONNECTIONLOSS" error...maybe they need to be in the SetUp due to the way the testing framework works? I'm not too sure since I didn't write this part. Does anyone else has some insight into this?